corehq/apps/case_search/templates/case_search/case_search.html

```diff
@@ -87,26 +87,18 @@ <h3>{% trans "Search Parameters" %}</h3>
                 <strong>XPath Expression</strong>
               </div>
               <div class="panel-body">
-                <div class="form-group" style="width:100%">
-                  <textarea style="min-width: 50%" spellcheck="false" class="form-control vertical-resize" name="xpath" type="text" placeholder="e.g: name = 'tyrion' or nickname='the imp'" data-bind="value: xpath"> </textarea>
+                <div class="form-group" style="width:100%" data-bind="foreach: xpath_expressions" >
+                  <textarea style="min-width: 50%" spellcheck="false" class="form-control vertical-resize"
+                    name="xpath" type="text" placeholder="e.g: name = 'tyrion' or nickname='the imp'" data-bind="value: xpath"> </textarea>
+                  <span class='btn btn-danger' data-bind="click: $parent.removeXPath">
+                    <i class="fa-regular fa-trash-can"></i> {% trans "Delete" %}
+                  </span>
                 </div>
+                <button class="btn btn-default" id="add-xpath" data-bind="click: addXPath">
+                  <i class="fa fa-plus"></i> {% trans "Add XPath" %}
+                </button>
               </div>
             </div>
-            {% if query_additions %}
```

Comment:
Just noting that references to this were removed here: https://github.com/dimagi/commcare-hq/pull/28278